govcd/catalog_test.go

```diff
@@ -84,3 +96,179 @@ func (vcd *TestVCD) Test_DeleteCatalog(check *C) {
 	check.Assert(catalog, Equals, Catalog{})
 
 }
+
```

Comment:
Please add a comment to each test case to explain the feature it is validating.

Reply:
Added.